quinn-proto/src/connection/mod.rs

```diff
@@ -717,6 +719,22 @@ where
         pad
     }
 
+    /// Returns the destination IP used in outgoing transmit packet
+    ///
+    /// Returns `None` if no specific IP should be used
+    fn transmit_dst_ip(&self) -> Option<IpAddr> {
```

Comment:
I think I'd prefer it like this:

```rust
match self.server_config.as_ref()?.use_initial_local_ip_on_send {
    true => self.local_ip,
    false => None,
}
```

Any thoughts?

Reply:
Definitely preferable. Will update